src/Directory.Build.props

```diff
@@ -39,6 +39,14 @@
     <UpdateXlfOnBuild>true</UpdateXlfOnBuild>
     <ErrorOnOutOfDateXlf>true</ErrorOnOutOfDateXlf>
 
+    <UpdateXlfOnBuild Condition="'$(CIBuild)' != 'true'">true</UpdateXlfOnBuild>
```

Comment:
Why is this coming in now?

Reply:
I think this won't have the desired effect because of https://github.com/Microsoft/msbuild/pull/3048/files/85cd086eefce5854d3f591ea67372aad6990d35d#r172350635

Reply:
Copy & paste error